build/containers/archive-analysis/Containerfile

```diff
@@ -24,7 +24,7 @@ COPY --from=build /build /build
 COPY build/containers/archive-analysis/root /
 
 RUN dnf install -y --setopt=install_weak_deps=False --setopt=tsflags=nodocs \
-        python3 procps-ng gettext /build/*.rpm valkey grafana grafana-pcp && \
+        python3 python3-pcp procps-ng gettext /build/*.rpm valkey grafana grafana-pcp && \
```

Comment:
Just noticed "/build/*.rpm" here - I think this Containerfile should to be updated to not build its own PCP nowadays (see the RUN commands on line 10 and 11 - turf these) but rather use PCP from the distribution.
This build-it-locally I think harks back to the times we were still building pmproxy, pmseries, et al, and is no longer needed.